packages/rx_bloc_cli/example/README.md

```diff
@@ -268,6 +268,8 @@ If there are new keys added to the main translation file they can be propagated
 
 Upon rebuild, your translations are auto-generated inside `lib/assets.dart`. In order to use them, you need to import the `l10n.dart` file from `lib/l10n/l10n.dart` and then access the translations from your BuildContext via `context.l10n.someTranslationKey` or `context.l10n.featureName.someTranslationKey`.
 
+Localization lookups are also supported. That means that you can request any remote localizations from a dedicated translations endpoint at app start. Grabbing any remote localization for existing features will replace the local translations with new ones.
```

Comment:
Please add documentation regarding the expected json structure either in the main README.md file or in a sperate file.